style/style.scss

```diff
@@ -0,0 +1,82 @@
+$main-bg: #141d26;
+
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    background: $main-bg;
+}
+
+.keyboard {
+    text-align: center;
+    margin: 0 auto;
+    width: 800px;
+    padding: 13px 0 13px 6px;
```

Comment:
there is an common convention to use even number values in css, so as for me it's better to follow this convention in future